lib/NormalModule.js

```diff
@@ -154,7 +154,9 @@ class NormalModule extends Module {
 				this.errors.push(new ModuleError(this, error));
 			},
 			exec: (code, filename) => {
+				// @ts-ignore Argument of type 'this' is not assignable to parameter of type 'Module'.
```

Comment:
normal module is not precisely implementing native module. This is why ts ignore directive is here. 

We can use jsdoc @implements to actually resolve it but then this class needs some additional members to satisfy normal module.

Reply:
This should not block merging this pull request though..